pkg/asset/templates/content/bootkube/etcd-host-service-endpoints.go

```diff
@@ -9,29 +9,29 @@ import (
 )
 
 const (
-	kubeSystemSecretEtcdSignerFileName = "kube-system-secret-etcd-signer.yaml.template"
+	etcdHostServiceEndpointsFileName = "etcd-host-service-endpoints.yaml.template"
 )
 
-var _ asset.WritableAsset = (*KubeSystemSecretEtcdSigner)(nil)
+var _ asset.WritableAsset = (*EtcdHostServiceEndpoints)(nil)
 
-// KubeSystemSecretEtcdSigner is the constant to represent contents of kube-system-secret-etcd-signer.yaml.template file.
-type KubeSystemSecretEtcdSigner struct {
+// EtcdHostServiceEndpoints is the constant to represent contents of etcd-service-endpoints.yaml.template file.
```

Comment:
Path does not match `etcdHostServiceEndpointsFileName`.  Again, I'd rather go with comments that just translate the name into English ("etcd service endpoints"?  Not sure what the host-yness is about) instead of having a copy/pasted-ish filename.

Reply:
> Path does not match `etcdHostServiceEndpointsFileName`. Again, I'd rather go with comments that just translate the name into English ("etcd service endpoints"? Not sure what the host-yness is about) instead of having a copy/pasted-ish filename.

host is for host-network